src/app/pages/storage/volumes/datasets/dataset-form/dataset-form.component.ts

```diff
@@ -285,19 +285,19 @@ export class DatasetFormComponent implements Formconfiguration{
     },
     {
       type: 'select',
-      name: 'exec',
-      placeholder: T('Exec'),
-      tooltip: T('Choose <b>On</b> or <b>Off</b>.'),
+      name: 'readonly',
+      placeholder: T('Read-only'),
+      tooltip: T('Choose if the dataset can be modified.'),
       options: [
         { label: 'On', value: 'ON' },
         { label: 'Off', value: 'OFF' }
       ],
     },
     {
       type: 'select',
-      name: 'readonly',
-      placeholder: T('Read-only'),
-      tooltip: T('Choose if the dataset can be modified.'),
+      name: 'exec',
+      placeholder: T('Exec'),
+      tooltip: T('Choose <b>On</b> or <b>Off</b>.'),
```

Comment:
Suggested exact wording:
`tooltip: T('Set whether processes can be executed from within this dataset.`),`